src/bootstrap/test.rs

```diff
@@ -1424,7 +1424,13 @@ note: if you're sure you want to do this, please open an issue as to why. In the
 
         cmd.arg("--src-base").arg(builder.src.join("tests").join(suite));
         cmd.arg("--build-base").arg(testdir(builder, compiler.host).join(suite));
-        cmd.arg("--sysroot-base").arg(builder.sysroot(compiler));
+
+        let sysroot = if builder.top_stage == 0 {
+            builder.initial_sysroot.clone()
```

Comment:
please leave a comment here explaining why it's necessary - i think you needed the paths in the standard library to match or something, right?

Reply:
Added a short comment.